bot/main.py

```diff
@@ -379,6 +379,22 @@ async def profile_error(ctx, error):
   await utils.send_failed_msg(ctx.channel, "!profile requires a SteamID [STEAM_0:0:11101] or @user argument")
   return
 
+@bot.command
+async def getsteamid(ctx, url: str):
+
+    # Check if URL links to the Steam Community, is a decimal value and within the length limit
+    if ('https://steamcommunity.com/profiles/' not in url) or \
+            (not url.split('/')[-1].isdecimal()) or \
+            (not len(url.split('/')[-1]) == 17):
```

Comment:
This could be done easier by the use of regex, take a look at how I made the regex for the steam id:

```def is_steamid(steamid): return re.search("^STEAM_[0-5]:[01]:\d+$", steamid)```